### DISCUSSION TITLE
Need some hints...

### DISCUSSION THREAD

**Discussion by awandahl** [mbridak/not1mm#142] [General]
I'm playing around with some functions and have a few observations/questions:

- no log entries are put in the log window, but they are visible in QSO history after restart of the program. Would have expected to see them after every QSO.
- no green arrow indicating my VFO fq in the BandMap. I run rigctld to my QMX transceiver and the VFO window indicates the correct fq
- bandmap doesn't change with band changes
- no hits in the CheckPartial from any source. I have updated the CTY and MASTER.SCP 

aw@5x7w-t2:~$ ls -la .local/lib/python3.10/site-packages/not1mm/data/MASTER.SCP
-rw-rw-r-- 1 aw aw 363257 Sep 18 16:51 .local/lib/python3.10/site-packages/not1mm/data/MASTER.SCP

I'm sure there is a very simple solution but I'm kind of stuck here... any help would be nice!

73 Anders SM0HPL  5X7W

**Comment by mbridak** [comment]
@awandahl Welp, not sure what functions you were playing around with. But that functionality does work out of the box. Here's a screen shot with it getting hits from all 3 sources.

![Screenshot from 2024-09-19 08-51-00](https://github.com/user-attachments/assets/b9d1dc06-013c-4dd4-9b5f-381540e6fbe4)

If I had to guess, you may have messed around with the multicast stuff and now the pieces can't talk to each other. That would explain why nothing updates for you. And it's needed for everything.

Mike K6GTE

**Reply by awandahl** [reply]
Thanks Mike! I think a complete reinstall is what I need... I haven't intentionally messed up things ;)

**Reply by awandahl** [reply]
A beautiful piece of software you have made, Mike! Now everything works on a brand new Armbian Ubuntu 24.04. I think pyQT6 wasn't properly installed on my 22.04 Jammy. Thanks!

**Comment by awandahl** [comment]
Revisiting this discussion... I had not1mm up and working with multicast and all, but then I changed my Ubuntu 24.04 Armbian from GNOME to Xfce4 and all of a sudden it doesn't work any longer. Is this a pure coincidence or can this change be the culprit. I have checked so multicast is activated and also  enabled IP forwarding like net.ipv4.ip_forward = 1  Follow up questions: in the example for installation of not1mm for Ubuntu 24.04 pip is used rather than pipx. Is it a reason behind that? I find pipx very swift and nice.

**Comment by mbridak** [comment]
It should make no difference which DE you are running at the moment. I hop between Gnome and XFCE all the time.
I've never had the multicast interface randomly stop working.

pip vs pipx: I wrote the pip example before I started to generally using pipx on the regular.
I'd recommend everyone use pipx. Or even uv. uv is faster still.

Someday I'll figure out flatpak, and it'll be easier still.


![image](https://github.com/user-attachments/assets/c894faab-86b0-4b19-939d-5dabe6435734)

**Reply by awandahl** [reply]
Why didn't I think about this before...?
sudo ufw allow in proto udp to 224.0.0.0/4
sudo ufw allow out proto udp to 224.0.0.0/4
